docs/en/week14/14-1.md

```diff
@@ -0,0 +1,185 @@
+---
+lang-ref: ch.14-1
+title: Deep Learning for Structured Prediction
+authors: Junrong Zha, Muge Chen, Rishabh Yadav, Zhuocheng Xu
+date: May 10, 2020
+---
+# Deep Learning for Structured Prediction
+<!-- RISHABH --------->
+## Structured Prediction
+### Overview
+It is the problem of predicting variable y for a given input x which is mutually dependent and constrained rather than scalar discrete or real values. The output variable does not belong to a single category but can have exponential or infinite possible values. 
+For eg: In case of speech/handwriting recognition or natural language translation, the output needs to be grammatically correct and it is not possible to limit the number of output possibilities. The task of the model is to capture the sequential, spatial, or combinatorial structure in the problem domain.
+
+### Early works on structured prediction
+This vector is fed to a TDNN which gives a feature vector which in case of model systems can bee compared to softmax that represents a category. One problem that arises in the case of recognizing the word that was pronounced is different people can pronounce the same word in different ways and speed. To solve this Dynamic Time Warping is used. 
+
+The idea is to provide the system with a set of pre-recorded templates that correspond to sequence or feature vectors that were recorded by someone. The neural network is trained at the same time as the template so that the system learns to recognize the word for different pronunciations. The latent variable allows us to timewarp the feature vector so as to match the length of the templates.
+
+<img src="https://i.imgur.com/T7ajVO7.png" style="zoom:100%;"/>
```

Comment:
This is not pointing to the image on the repo. Please, fix.
Read the instructions!!! https://atcold.github.io/pytorch-Deep-Learning/en/about/